src/components/BookItem.js

```diff
@@ -11,9 +11,9 @@ const BookItem = ({
       <h1>{title}</h1>
       <p>{author}</p>
       <div className="meta">
-        <button type="button">Comments</button>
+        <button className="button-unique" type="button">Comments</button>
```

Comment:
- [ ] Please add the vertical line as on design

Yours:
![image](https://github.com/0sugo/bookstore/assets/62705947/654a2715-e82b-4b0a-b4bd-79310aaea8dd)

Design:
![image](https://github.com/0sugo/bookstore/assets/62705947/0c2fc6e7-7116-4f23-96ed-1b3ca03acd05)

Reply:
- [ ] your design is missing the vertical line between the progress section and the current chapter section, kindly add
zeplin design :point_down: 
![image](https://github.com/0sugo/bookstore/assets/66097371/91bbb0e7-be45-4dbb-a9ee-69bed9e341f1)